### ISSUE TITLE
Better Forum

### ISSUE THREAD

**Issue by MaterArc** [LLK/scratch-www#7458]
<img width="320" alt="Screen Shot 2023-03-16 at 6 01 32 PM" src="https://user-images.githubusercontent.com/105017592/225762477-5fa09440-923e-4352-b3f6-77731812bb8c.png">

Maybe the forum could be changed to Questions About Scratch, as that is more relevant here.

**Comment by Adzboy** [comment]
Could show both but specify that HwS is only for projects whereas QaS is more general

**Comment by MaterArc** [comment]
Yeah!

**Comment by Racecar-Roaster** [comment]
Yeah. HwS is only for making code. However QaS is more about other things about scratch. They could add it or replace HwS.
